pages/common/dvc-dag.md

```diff
@@ -0,0 +1,20 @@
+# dvc dag
+
+> Visualize the pipeline(s) in dvc.yaml.
+> More information: <https://dvc.org/doc/command-reference/dag>.
+
+- Visualize entire pipeline:
+
+`dvc dag`
+
+- Visualize pipeline stages up to a target stage:
+
+`dvc dag {{target}}`
+
+- Export pipeline in dot format:
+
+`dvc dag --dot > pipeline.dot`
+
+- Export pipeline as SVG:
```

Comment:
This example seems to  be the same as the last, just exporting from graphviz `dot` in SVG. We generally try to avoid demonstrating the functionality of other commands in a tldr page. Let us remove this example.